frontend/src/components/RecipeList.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+
+const recipeAPI = axios.create({
+  baseURL: '/api/',
+});
+
+function RecipeList() {
+  const [recipes, setRecipes] = useState();
+
+  useEffect(() => {
+    async function getRecipeList() {
+      try {
+        const response = await recipeAPI.get('recipes');
+        setRecipes(response.data);
+      } catch (err) {
+        console.error('Error during api call:', err);
+      }
+    }
+    getRecipeList();
+  }, []);
+
+  return (
+    <>
+      <ul className="list-group">
+        {recipes ? (
+          recipes.map((recipe) => (
+            <li
+              key={recipe.id}
+              className="list-group-item list-group-item-action"
+            >
+              <Link to={`/recipes/${recipe.id}`} key={recipe.id}>
+                {recipe.name}
+              </Link>
+            </li>
+          ))
+        ) : (
+          <div>'Loading List...' </div>
```

Comment:
Így most ez megint az ul-en belül van... Valami ilyesmit is lehet írni:
```
{recipes ? (
    <ul className="list-group">
    {recipes.map((recipe) => (
    ...
    )}
    </ul>
) : 
(...)
}
```

Reply:
{recipes ? (
        <ul className="list-group">
          {recipes.map((recipe) => (
            <li
              key={recipe.id}
              className="list-group-item list-group-item-action"
            >
              <Link to={`/recipes/${recipe.id}`} key={recipe.id}>
                {recipe.name}
              </Link>
            </li>
          ))}{' '}
        </ul>
      ) : (
        <div>'Loading List...' </div>
      )}


Kész

Reply:
Hol? :)

Reply:
uh most nézem a ctrl + c nem működött hibátlanuk

`return (
    <>
      {recipes ? (
        <ul className="list-group">
          {recipes.map((recipe) => (
            <li
              key={recipe.id}
              className="list-group-item list-group-item-action"
            >
              <Link to={`/recipes/${recipe.id}`} key={recipe.id}>
                {recipe.name}
              </Link>
            </li>
          ))}{' '}
        </ul>
      ) : (
        <div>'Loading List...' </div>
      )}
    </>
  );`

Reply:
De ne ide írd commentben a kódot, hanem commitold föl :D